sosw/components/helpers.py

```diff
@@ -394,6 +395,30 @@ def validate_date_from_something(d):
     return validate_datetime_from_something(d).date()
 
 
+def validate_date_timestamp_from_something(d):
+    """
+    Converts the input `d` to timestamp.
+
+    :param d: Some input. Supported types:
+                * datetime.datetime
+                * datetime.date
+                * int - Epoch or Epoch milliseconds
+                * float - Epoch or Epoch milliseconds
+                * str (YYYY-MM-DD)
+                * str (YYYY-MM-DD HH:MM:SS)
+                * str(epoch time seconds as string)
+                * str(epoch time seconds (float) as string)
+    :return: Transformed `d`
+    :rtype: timestamp
```

Comment:
remove the `:rtype:` please and add it to the typehint in the function declaration as `... -> float:`